packages/e2e-tests/specs/plugins/plugins-api.test.js

```diff
@@ -76,4 +76,12 @@ describe( 'Using Plugins API', () => {
 			expect( pluginSidebarClosed ).toBeNull();
 		} );
 	} );
+
+	describe( 'Document Setting Custom Panel', () => {
+		it( 'Should render a custom panel inside Document Setting sidebar', async () => {
+			await openDocumentSettingsSidebar();
+			const pluginPostStatusInfoText = await page.$eval( '.edit-post-sidebar .my-document-setting-plugin', ( el ) => el.innerText );
```

Comment:
`pluginPostStatusInfoText` should be renamed. I don't think this test is solid enough as it didn't catch the issue with the wrong fill component defined.